Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -7748,8 +7748,8 @@
         <On Terms="Upgrade.AP_RoachCorpser.Add" Send="Destroy"/>
         <On Terms="ActorCreation; ValidateUnit AP_HaveRoachCorpser" Send="Create AP_RoachCorpser"/>
         <On Terms="ActorCreation; ValidateUnit AP_HaveRoachCorpser" Send="Destroy"/>
-        <On Terms="Upgrade.AP_TunnelingClaws.Add" Send="ModelSwap RoachUpgrade"/>
-        <On Terms="Upgrade.AP_TunnelingClaws.Add; ValidateUnit IsBuried" Send="AnimGroupApply Burrow Burrow Unburrow ClosingFull,Instant"/>
+        <On Terms="Upgrade.AP_HotSTunnelingClaws.Add" Send="ModelSwap RoachUpgrade"/>
+        <On Terms="Upgrade.AP_HotSTunnelingClaws.Add; ValidateUnit IsBuried" Send="AnimGroupApply Burrow Burrow Unburrow ClosingFull,Instant"/>

```

Comment:
This should be disabled (or checked against strains) as it could override strain model